currency/internal/cmd/main.go

```diff
@@ -0,0 +1,158 @@
+package main
```

Comment:
Can this be done with go generate (or is there reason not to, I don't know the context well enough, or know go generate well enough)?